datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTab.tsx

```diff
@@ -70,6 +136,11 @@ export const SchemaTab = ({ properties }: { properties?: any }) => {
                 hasKeySchema={hasKeySchema}
                 showKeySchema={showKeySchema}
                 setShowKeySchema={setShowKeySchema}
+                lastUpdatedTimeString={lastUpdatedTimeString}
+                selectedVersion={selectedVersion}
+                semanticVersionList={semanticVersionList}
```

Comment:
since we say "selectedVersion" and not "selectedSemanticVersion", the prop called "semanticVersionList" can really just be "versionList" IMO. I like to keep the names consistent where possible

Reply:
Agreed!